app/frontend/wallet/wallet.ts

```diff
@@ -0,0 +1,95 @@
+import BlockchainExplorer from './blockchain-explorer'
+import {AccountManager} from './account-manager'
+
+const Wallet = ({config, cryptoProvider}) => {
+  const blockchainExplorer = BlockchainExplorer(config)
+
+  const accountManager = AccountManager({config, cryptoProvider, blockchainExplorer})
+
+  function isHwWallet() {
+    return cryptoProvider.isHwWallet()
+  }
+
+  function getWalletName() {
+    return cryptoProvider.getWalletName()
+  }
+
+  function submitTx(signedTx): Promise<any> {
+    const params = {
+      walletType: getWalletName(),
+      // TODO: stakeKey
```

Comment:
this comment doesn't seem very helpful to a casual reader/future ourselves - why the stakeKey would be needed here and what's the impact of not having it there right now?

Reply:
@xdzurman